Lib/multiprocessing/managers.py

```diff
@@ -1195,6 +1195,38 @@ def __ior__(self, value):
 
 collections.abc.MutableMapping.register(_BaseDictProxy)
 
+_set_proxy_methods = {
+    '__and__', '__class_getitem__', '__contains__', '__iand__', '__ior__',
+    '__isub__', '__iter__', '__ixor__', '__len__', '__or__', '__rand__',
+    '__ror__', '__rsub__', '__rxor__', '__sub__', '__xor__',
+    '__ge__', '__gt__', '__le__', '__lt__',
+    'add', 'clear', 'copy', 'difference', 'difference_update', 'discard',
+    'intersection', 'intersection_update', 'isdisjoint', 'issubset',
+    'issuperset', 'pop', 'remove', 'symmetric_difference',
+    'symmetric_difference_update', 'union', 'update',
+}
+_set_proxy_methods = sorted(_set_proxy_methods)
+_BaseSetProxy = MakeProxyType("_BaseSetProxy", _set_proxy_methods)
+del _set_proxy_methods
```

Comment:
```suggestion
_BaseSetProxy = MakeProxyType("_BaseSetProxy", (
    '__and__', '__class_getitem__', '__contains__', '__iand__', '__ior__',
    '__isub__', '__iter__', '__ixor__', '__len__', '__or__', '__rand__',
    '__ror__', '__rsub__', '__rxor__', '__sub__', '__xor__',
    '__ge__', '__gt__', '__le__', '__lt__',
    'add', 'clear', 'copy', 'difference', 'difference_update', 'discard',
    'intersection', 'intersection_update', 'isdisjoint', 'issubset',
    'issuperset', 'pop', 'remove', 'symmetric_difference',
    'symmetric_difference_update', 'union', 'update',
))
```

We no more need the sorted etc since we're hardcoding everything now.